lib/services/address/index.js

```diff
@@ -154,20 +155,17 @@ AddressService.prototype.getAPIMethods = function() {
  * Called by the Bus to get the available events for this service.
  */
 AddressService.prototype.getPublishEvents = function() {
-  return [
-    {
-      name: 'address/transaction',
-      scope: this,
-      subscribe: this.subscribe.bind(this, 'address/transaction'),
-      unsubscribe: this.unsubscribe.bind(this, 'address/transaction')
-    },
-    {
-      name: 'address/balance',
-      scope: this,
-      subscribe: this.subscribe.bind(this, 'address/balance'),
-      unsubscribe: this.unsubscribe.bind(this, 'address/balance')
-    }
-  ];
+  return [{
```

Comment:
Let's keep style changes separate please. Also, the majority of the code base is in the previous style.